CHANGELOG.md

```diff
@@ -6,6 +6,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## Version 4.2.1
+
+This release contains a low-severity security related fix. Users of the `4.x` series of
+releases are advised to update.
+
+Note that this release contains a breaking change since the API of
```

Comment:
Might wanna add an explanation why we don't release a major version since it is a breaking change